package.json

```diff
@@ -36,9 +36,9 @@
     "clean": "rimraf dist"
   },
   "dependencies": {
-    "@patternfly/patternfly": "4.132.2",
-    "@patternfly/react-core": "4.152.4",
-    "@patternfly/react-table": "4.29.58",
+    "@patternfly/patternfly": "4.151.4",
```

Comment:
update so i can use the newer `--pf-c-backdrop--Position` variable, and also match insights-chrome
https://github.com/RedHatInsights/insights-chrome/blob/65b1408ebbcc29edc548e0fe7043542889fe1040/package.json#L136